src/commands/build/category.txt

```diff
@@ -0,0 +1 @@
+TODO: Build command category description
```

Comment:
@bryukov @lukekim Here's my proposal 
* mobile-center build * Commands for queueing builds, getting build status and downloading binaries and logs for completed builds
* mobile-center build queue* Queue a new build
* mobile-center build branches * Get information about branches and their latest build status
* mobile-center build download * Download the binary, logs or symbols for a completed build